packages/@react-spectrum/table/test/TableSizing.test.js

```diff
@@ -0,0 +1,857 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+jest.mock('@react-aria/live-announcer');
+import {act, render as renderComponent, within} from '@testing-library/react';
+import {ActionButton} from '@react-spectrum/button';
+import Add from '@spectrum-icons/workflow/Add';
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '../';
+import {HidingColumns} from '../stories/HidingColumns';
+import {Provider} from '@react-spectrum/provider';
+import React from 'react';
+import {theme} from '@react-spectrum/theme-default';
+import userEvent from '@testing-library/user-event';
+
+let columns = [
+  {name: 'Foo', key: 'foo'},
+  {name: 'Bar', key: 'bar'},
+  {name: 'Baz', key: 'baz'}
+];
+
+let nestedColumns = [
+  {name: 'Test', key: 'test'},
+  {name: 'Tiered One Header', key: 'tier1', children: [
+    {name: 'Tier Two Header A', key: 'tier2a', children: [
+      {name: 'Foo', key: 'foo'},
+      {name: 'Bar', key: 'bar'}
+    ]},
+    {name: 'Yay', key: 'yay'},
+    {name: 'Tier Two Header B', key: 'tier2b', children: [
+      {name: 'Baz', key: 'baz'}
+    ]}
+  ]}
+];
+
+let items = [
+  {test: 'Test 1', foo: 'Foo 1', bar: 'Bar 1', yay: 'Yay 1', baz: 'Baz 1'},
+  {test: 'Test 2', foo: 'Foo 2', bar: 'Bar 2', yay: 'Yay 2', baz: 'Baz 2'}
+];
+
+
+let manyItems = [];
+for (let i = 1; i <= 100; i++) {
+  manyItems.push({id: i, foo: 'Foo ' + i, bar: 'Bar ' + i, baz: 'Baz ' + i});
+}
+
+describe('TableViewSizing', function () {
```

Comment:
Why moved?

Reply:
Effort to make test file smaller, it was getting slow for Jest to run
It may be that Jest sharding takes care of this, however, it was also a bit unwieldy in terms of human readability, scrolling that much just causes me to want to die, and this new file is where I'll be adding a bunch of tests around resize behavior in my next PR